src/lib/utils/queries.ts

```diff
@@ -0,0 +1,22 @@
+import { queries, ParamsType } from '@/data/queries'
+
+jest.mock('')
```

Comment:
this jest mock feels out-of-place here

Reply:
😂 Just a bit. I'm not sure how that got there. I had to introduce a mock in the test file for this code, so it's not a complete shock, but I clearly started typing in the wrong place.